src/components/snackbar/snackbar.scss

```
@@ -1,3 +1,5 @@
+@forward '../portal/partial-styles/portal-container.scss';
```

Stylelint (17.3.0)

[error] Unexpected extension ".scss" in `@forward` (scss/load-partial-extension)

(scss/load-partial-extension)